### PR TITLE
feat(Workspaces): allow admin to regenerate db password

### PR DESCRIPTION
**Pull request by cheikhgwane**
Allow workspace admin to regenerate a new password for the dedicated workspace database. 

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Add button for displaying regenerate password dialog
- Integrate mutation for generating a new  database password

## Screenshots / screencast

image

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -1264,6 +1279,7 @@ type Mutation {
   createConnection(input: CreateConnectionInput!): CreateConnectionResult!
   updateConnection(input: UpdateConnectionInput!): UpdateConnectionResult!
   deleteConnection(input: DeleteConnectionInput!): DeleteConnectionResult!
+  generateDatabaseNewPassword(input: GenerateDatabaseNewPasswordInput!): GenerateDatabaseNewPasswordResult!
```
Comment (qgerome) [inline schema.graphql:1282]: WDYT of `generateNewDatabasePassword` ?

Comment (qgerome) [reply schema.graphql:1282]: Ha, it's already merged in the backend.. Too bad.

Comment (cheikhgwane) [reply schema.graphql:1282]: I was hesitating between the two but that's not a problem I can open another PR for that generateNewDatabasePassword (should be fast)

**Comment by qgerome** [comment]
No it’s ok, it’s just a nit pick.
Quentin Gérôme
+32 471 490 190

**Comment by pvanliefland** [comment]
I agree that `generateNewDatabasePassword` is better, if it doesn't take too much time let's go for it

File: src/workspaces/features/GenerateDatabasePasswordDialog/GenerateDatabasePasswordDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          {t("Generate")}
+        </Button>
+      </Dialog.Actions>
+    </Dialog>
+  );
+};
+
+GenerateWorkspaceDatabasePasswordDialog.fragments = {
+  workspace: gql`
+    fragment GenerateWorkspaceDatabasePassword on Workspace {
```
Comment (qgerome) [inline src/workspaces/features/GenerateDatabasePasswordDialog/GenerateDatabasePasswordDialog.tsx:76]:
```suggestion
    fragment GenerateWorkspaceDatabasePasswordDialog_workspace on Workspace {
```

**Review by qgerome — CHANGES_REQUESTED** [review]

**Comment by qgerome** [comment]
Could you align vertically the breadcrumbs like on the other pages
CleanShot 2023-03-06 at 12 10 00@2x

If the server responds with a success status, it would be nice to display an alert to the user to notify him that it worked as expected.

**Pull request merged**